### DISCUSSION TITLE
Instructions for Synology NAS

### DISCUSSION THREAD

**Discussion by EatingTheElephant** [sickcodes/Docker-OSX#842] [General]
Hello, I have a Synology 423+. I am trying to get this running. I'm very inexperienced in this realm. Are there instructions on how to get this running in Synology Container Manager? It is in the Registry, and I was able to download the image, but that's all I could do. I tried to create a project, but it didn't work. I also tried to run the image but couldn't get it working either. Any advice would be appreciated.

**Comment by ralphdt** [comment]
Try adding Portainer and using Stacks - Web Editor to add your build parameters
